stripe.go

```diff
@@ -44,6 +45,11 @@ const TotalBackends = 2
 // one from `uname`.
 const UnknownPlatform = "unknown platform"
 
+// maxNetworkRetryDelay and minNetworkRetryDelay defines sleep time in milliseconds between
+// tries to send HTTP request again after network failure.
+const maxNetworkRetryDelay = 5000
+const minNetworkRetryDelay = 500
```

Comment:
So I think just for maximum correctness and because Go supports this natively, we should use `time.Duration` for both of these. So something like `5000 * time.Millisecond`.